types/web3/utils.d.ts

```diff
@@ -58,7 +58,8 @@ export default interface Utils {
     fromAscii(val: string): string;
     fromDecimal(val: string | number | BigNumber): string;
     fromUtf8(val: string): string;
-    fromWei(val: string | number | BigNumber, unit: Unit): string | BigNumber;
+    fromWei(val: BigNumber, unit?: Unit): BigNumber;
+    fromWei(val: string | number, unit?: Unit): string;
```

Comment:
Please add a unit test for this.

Reply:
@Levino @phra 

Tests are added, thanks.